zebra-network/src/peer_set/initialize.rs

```diff
@@ -40,14 +41,7 @@ pub async fn init<S>(
     config: Config,
     inbound_service: S,
 ) -> (
-    impl Service<
-            Request,
-            Response = Response,
-            Error = BoxedStdError,
-            Future = impl Future<Output = Result<Response, BoxedStdError>> + Send,
-        > + Send
-        + Clone
-        + 'static,
+    Buffer<BoxService<Request, Response, BoxedStdError>, Request>,
```

Comment:
Can we make this a publicly exported type alias?

Reply:
I'm not opposed to it but I don't think it adds much beyond `Buffer<BoxService>`; I think it's useful to highlight that the returned service is already a `Buffer` (which is unlike other cases where we could have component crates just return a `BoxService`, whereas here the network needs access to the service to do self-maintenance)

Reply:
Seems like something we could revisit after we get some consistency between different zebra-* crates.

Reply:
sounds good,